docs/codeql/reusables/data-extensions.rst

```diff
@@ -0,0 +1,4 @@
+.. _data-extensions:
+
+Data Extensions
```

Comment:
Will this page be empty for now?

Reply:
I will add a short private beta statement that data extensions are documented in a google doc.